packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalDisclaimer/WithdrawalDisclaimer.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { InlineMessage } from '../../../../../../components/Base/InlineMessage';
+import './WithdrawalDisclaimer.scss';
+
+const WithdrawalDisclaimer = () => (
+    <div className='wallets-withdrawal-crypto-disclaimer'>
```

Comment:
v. minor: insonsistency - component name is WithdrawalWallets, so BEM name should be wallets-withdrawal-wallets (unless its crypto-specific - then whole component should be named WithdrawalCryptoDisclaimer)

Reply:
Will change this 👀 .

Reply:
Hi @wojciech-deriv,
Yes this is crypto-specific.

Reply:
so if its crypto specific, then whole component should be named WalletsWithdrawalCryptDisclaimer